core/src/integrationTest/java/tech/pegasys/eth2signer/core/DirectoryBackedArtifactSigningProviderIntegrationTest.java

```diff
@@ -127,6 +128,7 @@ void missingHashicorpFingerprintFileWhichCannotBeCreatedShowsUsefulErrorLog() th
       final Path filename = createFileWithContent(signingMetadata);
 
       configsDirectory.toFile().setWritable(false);
+      signerProvider.loadSigners();
```

Comment:
Is this needed now because we don't lazily load signers anymore? Can the loadSigners be moved into the setup method?

Reply:
Alas, it cannot be moved, as the config file(s) need to be created before loading.